devito/finite_differences/differentiable.py

```diff
@@ -54,6 +58,14 @@ def is_Staggered(self):
     def _fd(self):
         return dict(ChainMap(*[getattr(i, '_fd', {}) for i in self._args_diff]))
 
+    @cached_property
+    def _symbolic_functions(self):
+        return [i for i in list(self._functions) if i.coefficients == 'symbolic']
+
+    @cached_property
+    def _symbolic_coefficients(self):
```

Comment:
is this supposed to return the actual Coefficients, in the future?

Reply:
I'm not sure (was thinking about this), but probably not since they will generally be stored in the `Eq` object.

Reply:
then perhaps a "booleanyc" name would be better such as ``_uses_symbolic_coefficients``? 

or perhaps use ``_symbolic_functions`` directly?

Reply:
I've changed to `_uses_symbolic_coefficients` for now?